src/app/productConfig/productConfigForm/productConfigForm.component.js

```diff
@@ -210,6 +226,11 @@ class ProductConfigFormController {
     this.errorChangingFrequency = false
     const lastFrequency = this.productData.frequency
     this.productData.frequency = product.name
+
+    if (this.isBrandedCheckout) {
+      this.itemConfig.frequency = product.display
```

Comment:
Same question as before, was this something that has been broken in branded checkout for a while? Or did cover fees change the behavior of something?

Reply:
It seems when the initial implementation was done, just the bare minimum was brought into this branded checkout `itemConfig` object. It's not that something was broken, but that this value wasn't used, so it wasn't put in the object.